caliper_tracking/processor.py

```diff
@@ -21,3 +23,5 @@ def __call__(self, event):
     def send(self, event):
         if not event['event_type'].startswith('/'):
             logger.info(self.__call__(event))
+        else:
+            logger.info(link_event_transformer(event))
```

Comment:
This will work for all event types other than caliper specified? 

How many events has this been tested with?

Reply:
yes this will transform all page-view events into caliper-NavigationEvent.